mappings/net/minecraft/village/TradeOffer.mapping

```diff
@@ -7,7 +7,9 @@ CLASS net/minecraft/unmapped/C_argwryfu net/minecraft/village/TradeOffer
 	FIELD f_lgvzktct demandBonus I
 	FIELD f_mgalkhcf maxUses I
 	FIELD f_orryzsqt merchantExperience I
+	FIELD f_qxgedhuk ignoreTags Z
```

Comment:
```suggestion
	FIELD f_qxgedhuk ignoreNbt Z
```
I know this is referred to as "ignore_tags" in strings, but we call it nbt everywhere else anyway, so I feel this is more clear